abel/basex.py

```diff
@@ -51,40 +55,42 @@
 #############################################################################
 
 
-def basex_transform(data, nbf='auto', basis_dir='./', dr=1.0, verbose=True,
+def basex_transform(data, nbf='auto', reg=0.0, basis_dir='./', dr=1.0, verbose=True,
                     direction='inverse'):
-    """ 
-    This function performs the BASEX (BAsis Set EXpansion) 
-    Abel Transform. It works on a "right side" image. I.e., 
+    """
+    This function performs the BASEX (BAsis Set EXpansion)
+    Abel Transform. It works on a "right side" image. I.e.,
     it works on just half of a cylindrically symmetric
-    object and ``data[0,0]`` should correspond to a central pixel. 
-    To perform a BASEX transorm on 
+    object and ``data[0,0]`` should correspond to a central pixel.
+    To perform a BASEX transorm on
     a whole image, use ::
-    
+
         abel.Transform(image, method='basex', direction='inverse').transform
 
-    This BASEX implementation only works with images that have an 
+    This BASEX implementation only works with images that have an
     odd-integer width.
-    
+
     Note: only the `direction="inverse"` transform is currently implemented.
-    
+
 
     Parameters
     ----------
     data : a NxM numpy array
         The image to be inverse transformed. The width (M) must be odd and ``data[:,0]``
-        should correspond to the central column of the image. 
+        should correspond to the central column of the image.
     nbf : str or list
         number of basis functions. If ``nbf='auto'``, it is set to ``(n//2 + 1)``.
         *This is what you should always use*,
         since this BASEX implementation does not work reliably in other situations!
         In the future, you could use
         list, in format [nbf_vert, nbf_horz]
+    reg : float
```

Comment:
This needs a longer description about how this is the tikhonov regularization factor can smooth the data, and provide some estimate of how to select and appropriate value.